.github/workflows/test.yml

```
@@ -16,46 +16,46 @@ jobs:
-        path-to-profile: coverage.out
-        flag-name: Go-${{ matrix.go }}
-      env:
-        KUBEBUILDER_ASSETS: "$(pwd)/bin"
+      - name: Test
+        run: sudo PATH=${PATH}:./bin ./hack/test-go.sh
+
+      - name: Send coverage
+        uses: shogo82148/actions-goveralls@9606dbc5ac5cf888a0e9ef901515c3cd516a2790 # v1
+        with:
+          path-to-profile: coverage.out
+          flag-name: Go-${{ matrix.go }}
```

actionlint (1.7.12)

[error] property "go" is not defined in object type {os: string}

(expression)